src/Services/Page.vala

```diff
@@ -78,6 +86,22 @@ public class ENotes.PageTable : DatabaseTable {
         }
 
         set_table_name ("Page");
+        /// @translator this code summarises a notebook page. Instead of given youtube link this code changes into Youtube Video      
+    
+        // Explaination for link: Regex for [Something](Something). As greedy as editor on markdown
+        var link = new BLMember(/\[[a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\]\([a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\)/, _("Link"));
+       
+        //  \[\^\d+\]:? leads to e.g. [^32], [^68]: 
+        var anchor = new BLMember (/\[\^\d+\]:?/, "");
+    
+        regex_complex_commands.add (link);
+        regex_complex_commands.add (anchor);
+              
+        var plugin_blacklist_member = PluginManager.get_instance ().get_all_blacklist_members ();
```

Comment:
Can't you just do `regex_complex_commands = PluginManager.get_instance ().get_all_blacklist_members ();` and then add the link and anchor regexes? :) 

It would avoid that extra foreach on the bottom